### PR TITLE
ZEPPELIN-2195. Use PYSPARK_PYTHON and PYSPARK_DRIVER_PYTHON over zeppelin.pyspark.python

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
`zeppelin.pyspark.python` is zeppelin configuration for the python exec on driver side, it won't affect executor side. It would be better to use `PYSPARK_PYTHON` and `PYSPARK_DRIVER_PYTHON`  which is what spark use officially.  So that user can define their own python exec in interpreter setting for different version of python rather than defining them `zeppelin-env.sh` which is shared globally. 

### What type of PR is it?
[ Improvement ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-2195

### How should this be tested?
Tested it manually. 

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Thanks for the improvement.

Do you think this change worth to be mentioned in the [upgrade.md](
https://github.com/apache/zeppelin/blob/master/docs/install/upgrade.md#upgrading-from-zeppelin-06-to-07) with new section "Upgrading from Zeppelin 0.7 to 0.8" ?

File: docs/interpreter/spark.md [hunk 1]
```diff
@@ -104,9 +104,16 @@ You can also set other Spark properties which are not listed in the table. For a
     <td>Local repository for dependency loader</td>
   </tr>
   <tr>
-    <td>zeppelin.pyspark.python</td>
+    <td>PYSPARK_PYTHON</td>
     <td>python</td>
-    <td>Python command to run pyspark with</td>
+    <td>Python binary executable to use for PySpark in both driver and workers (default is <code>python</code> if available, otherwise <code>python</code>).
```
Comment (felixcheung) [inline docs/interpreter/spark.md]: this is saying "(default is <code>python</code> if available, otherwise <code>python</code>)."
?

**Comment by zjffdu** [comment]
Thanks @Leemoonsoo @felixcheung for review. Comment is addressed.

**Comment by felixcheung** [comment]
merging if no more comment

**Pull request closed without merging**